benchmarks/ahi_hsd_benchmarks.py

```diff
@@ -41,15 +41,15 @@ def setup_cache(self):
             from satpy.demo import download_typhoon_surigae_ahi
             download_typhoon_surigae_ahi(channels=[1, 2, 3, 4], segments=[4])
         except ImportError:
-            assert len(get_filenames(self.subdir)) == 4
+            assert len(get_filenames(self.subdir)) == 4  # nosec
```

Comment:
Just curious, why do security checks flag this?

Reply:
because `assert`

Reply:
Ah asserts outside of tests. Ok. Got it.